framework/src/test/java/org/tron/common/storage/leveldb/LevelDbDataSourceImplTest.java

```diff
@@ -107,6 +109,13 @@ public void testReset() {
         Args.getInstance().getOutputDirectory(), "test_reset");
     dataSource.resetDb();
     assertEquals(0, dataSource.allKeys().size());
+    assertEquals("LEVELDB", dataSource.getEngine());
```

Comment:
Please add a test to cover this PR, do not add tests that is not relevant in this PR